doc/source/index.rst

```diff
@@ -85,8 +85,168 @@ should have.
+      resource_group=resource_group
+   )
+
+Configure parameters
+~~~~~~~~~~~~~~~~~~~~
+
+Let's keep everything in one place so it's easy to maintain.
+
+.. code-block:: python
+
+   name = "dask-azureml"
+
+   ### vnet settings
+   vnet_rg = ws.resource_group
+   vnet_name = "dask_azureml_vnet"
+   subnet_name = "default"
+
+   ### azure ml names: ct - compute target, env - environment
+   ct_name  = "dask-ct"
+   env_name = "AzureML-Dask-CPU"
+
+If you are running from an AzureML Compute Instance (Jupyter Lab) you should put both,
+the ``ComputeTarget`` (see `Compute Target <https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.compute.computetarget?view=azure-ml-py>`_ documentation page) 
+and the Compute Instance on the
+same `virtual network <https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview>`_.
+
+The ``AzureMLCluster`` class allows you to submit the job from your local machine as well (tested on DOS CLI and bash). In such case it is not necesary to 
+for the Compute Target to be on a virtual network (it is still preferred to hide the cluster in a virtual network). 
+However, you need to provide some administator name (other than ``admin``) and private and public SSH keys.
+
+.. code-block:: python
+
+   ### credentials
+   admin_username = name.split("-")[0]   ### dask
+   admin_ssh_key_pub = "<path-to-public-key>"
+   admin_ssh_key_priv = "<path-to-private-key>"
+
+The above credentials will be used to create an SSH tunnel between the headnode and your local machine
+so you can communicate with the Dask cluster.
+
+Create Compute Target
+~~~~~~~~~~~~~~~~~~~~~
+
+Next, let's create or retrieve already existing compute target. For a full list of
+VMs check here: `Windows <https://azure.microsoft.com/en-us/pricing/details/virtual-machines/windows/>`_ 
+and `Linux <https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/>`_.
+
+.. code-block:: python
+
+   # In this example, we will use ``STANDARD_DS12_V2`` VM because it is cheaper than others
+   vm_name = "STANDARD_DS12_V2"
+
+   ### UNCOMMENT BELOW FOR LOCAL RUNS
+   # with open(admin_ssh_key_pub, "r") as f:
+   #    ssh_key_pub = f.read().strip()
+
+   if ct_name not in ws.compute_targets:
+      # create config for Azure ML cluster
+      # change properties as needed
+      config = AmlCompute.provisioning_configuration(
+         vm_size=vm_name,
+         min_nodes=0,
+         max_nodes=2,
+         vnet_resourcegroup_name=vnet_rg,
+         vnet_name=vnet_name,
+         subnet_name=subnet_name,
+         idle_seconds_before_scaledown=300,
+
+         ### UNCOMMENT BELOW FOR LOCAL RUNS
+         # admin_username=admin_username,
+         # admin_user_ssh_key=ssh_key_pub,
+         # remote_login_port_public_access='Enabled',
+      )
+      ct = ComputeTarget.create(ws, ct_name, config)
+      ct.wait_for_completion(show_output=True)
+   else:
+      ct = ws.compute_targets[ct_name]
```

Comment:
At this point I got the following error.

```
Creating
AmlCompute wait for completion finished

Terminal state of "Failed" has been reached

Provisioning errors: [{'error': {'code': 'VNetNotFound', 'message': "Long running operation failed with status 'Failed'. Additional Info:'The specified subnet id /subscriptions/06cd8521-20cf-4eef-9f49-82f7345f53e6/resourceGroups/KDD-Group/providers/Microsoft.Network/virtualNetworks/dask_azureml_vnet/subnets/default is not found'", 'details': []}}]
```

I expect new users will not have a default vnet created and therefore this code example will also need to create one.

Reply:
This should have been resolved.